### ISSUE TITLE
Is there any convenient way to resume my training?

### ISSUE THREAD

**Issue by scut-salmon** [facebookresearch/maskrcnn-benchmark#558]
## ❓ Questions and Help
Hi, I think my question is simple, Just to resume my training from a checkpoint. The application scenarios is that I thought my training epoch is not enough, I want to train with the learning rate of this checkpoint for more epoch.

I think that maybe this project has provide some tools, so that I don't need to write the code by myself.

Any suggestion would be appreciated!

**Comment by fmassa** [comment]
Just rerun the training script pointing to the same OUTPUT_DIR as the other run, and change the number considerations in your config

**Comment by scut-salmon** [comment]
@fmassa Excuse me, I don't really get the meaning of "number considerations in your config". How can I tell the model to choose which checkpoint to resume?

**Comment by scut-salmon** [comment]
For example, my initial training scrip is 
python tools/train_net.py --config-file "configs/e2e_mask_rcnn_R_101_FPN_1x_my.yaml" SOLVER.IMS_PER_BATCH 2 SOLVER.BASE_LR 0.0025 SOLVER.MAX_ITER 180000 SOLVER.STEPS "(120000, 160000)" TEST.IMS_PER_BATCH 1 OUTPUT_DIR 'output/mtwi'
How should I resume if I want to retrain from 120000, and change my learning rate in 200000, 250000, 280000 epoch

**Comment by fmassa** [comment]
It was a typo, I meant number of iterations. 

If you want to change the checkpoint, then I recommend switching the OUTPUT_DIR, and pointing the MODEL.WEIGHT to the right checkpoint

**Comment by scut-salmon** [comment]
That's clear, Thanks

**Comment by scut-salmon** [comment]
@fmassa Hi, I try and find that's not I want. In my case, if I point the MODEL.WEIGHT to model_0120000.pth , the learning rate after 120000 epoch is 0.00025, however, I would like to retrain with learning rate 0.0025, How can I do it. My retrain script is as follow.

python tools/train_net.py --config-file "configs/e2e_mask_rcnn_R_101_FPN_1x_my.yaml" SOLVER.IMS_PER_BATCH 2 SOLVER.BASE_LR 0.0025 SOLVER.MAX_ITER 360000 SOLVER.STEPS "(240000, 320000)" TEST.IMS_PER_BATCH 1 OUTPUT_DIR 'output/mtwi_101_all_resume' MODEL.WEIGHT "output/mtwi_101_all/model_0120000.pth"

**Comment by fmassa** [comment]
@scut-salmon just create a new checkpoint file that has the same `model`, but deletes the `learning_rate_scheduler` maybe?

**Comment by BishwaBS** [comment]
1) Load the model instance again
model = modellib.MaskRCNN(mode="training", config=config,
                          model_dir=MODEL_DIR)
2) load the weights 
model.load_weights(path to your model saved in last epoch)

3) start training as before, it should work
